### PR TITLE
Fix usage of FormatUri method

### PR DESCRIPTION
**Pull request by dampir**
As part of my work on #1120 I've done some improvements in ApiUrls.cs class.

I've found out that not all clients use ApiUrls to build Uri and get right API endpoint. As far it is directly connected with my work, I fixed these cases.

- [x] Fix usage of "repos/{0}/{1}".FormatUri(owner, name)
- [x] Fix usage of "user/keys/{0}".FormatUri(id)
- [x] Fix usage of "users/{0}".FormatUri(login)
- [x] Fix usage of "repos/{0}/{1}/stats/contributors".FormatUri(owner, repositoryName)
- [x] Fix usage of "repos/{0}/{1}/stats/commit_activity".FormatUri(owner, repositoryName)
- [x] Fix usage of "repos/{0}/{1}/stats/code_frequency".FormatUri(owner, name)
- [x] Fix usage of "repos/{0}/{1}/stats/participation".FormatUri(owner, name)
- [x] Fix usage of "repos/{0}/{1}/stats/punch_card".FormatUri(owner, name)
- [x] Fix usage of "repos/{0}/{1}/collaborators/{2}".FormatUri(owner, repo, user)
- [x] Fix usage of "orgs/{0}/members/{1}".FormatUri(org, user)
- [x] Fix usage of "orgs/{0}".FormatUri(organizationName)

/cc @shiftkey, @ryangribble

### CODE REVIEW

**Comment by dampir** [comment]
Could anybody merge it if there is no problems?

/cc @shiftkey, @ryangribble

**Comment by ryangribble** [comment]
The builds failed after recent rebase?

**Comment by dampir** [comment]
@ryangribble yes and don't have any suspicions why it failed...

**Comment by ryangribble** [comment]
looks like `OrganizationMembersFilter` and `OrganizationMembersRole` are also missing

**Comment by dampir** [comment]
Hope it is fixed in the last commit. Merge/rebase always makes my cry :cry: 

https://media.giphy.com/media/EQIlRbpdee5P2/giphy.gif

File: Octokit.Reactive/Octokit.Reactive.csproj [hunk 1]
```diff
@@ -207,6 +207,10 @@
     </CodeAnalysisDictionary>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\Octokit\Octokit-Mono.csproj">
```
Comment (shiftkey) [inline Octokit.Reactive/Octokit.Reactive.csproj]: This reference isn't necessary

File: Octokit.Reactive/Clients/ObservableOrganizationMembersClient.cs [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ﻿using System;
+using System.Collections.Generic;
```
Comment (shiftkey) [inline Octokit.Reactive/Clients/ObservableOrganizationMembersClient.cs]: I'm not sure why these using statements are needed

**Comment by shiftkey** [comment]
@dampir a couple of unnecessary introduced changes, but aside from that this looks good. :heart: seeing the inlined URLs being extracted!

File: Octokit.Reactive/Octokit.Reactive.csproj (hunk 1 above)
Comment (dampir) [inline Octokit.Reactive/Octokit.Reactive.csproj]: Yep, fixed!

File: Octokit.Reactive/Clients/ObservableOrganizationMembersClient.cs (hunk 2 above)
Comment (dampir) [inline Octokit.Reactive/Clients/ObservableOrganizationMembersClient.cs]: I removed all unused usings in the file.

**Comment by dampir** [comment]
> seeing the inlined URLs being extracted!
Without it I can't properly generate new urls to access API by repo id :smile: 
Hope it would be merged as far as possible in order I can pull out these these new urls.

**Pull request merged**